scipy/spatial/meson.build

```diff
@@ -40,7 +40,7 @@ py3.extension_module('_qhull',
     'qhull_src/src'
   ],
   link_args: version_link_args,
-  dependencies: [np_dep],
+  dependencies: [lapack_dep, np_dep],
```

Comment:
This shouldn't be necessary, and removing it works for me locally. If it is needed for something, that'd be a potential problem with `cython_lapack` usage. Is this a left-over, or do you still see a need for this?

Reply:
Good catch! This was a holdover from my ILP64 experiments. `qhull_misc.h` includes `npy_cblas.h` for `CBLAS_INT`, which is a 64-bit integer if `HAVE_BLAS_ILP64` is defined. I plan to add this as an argument to `lapack_dep` and will reintroduce this change in a separate PR.